CIP-conway-keys/README.md

```diff
@@ -0,0 +1,245 @@
+| `cc_cold_xvk` | CIP-1852’s constitutional committee extended verification signing key | Ed25519 public key with chain code |
+| `cc_cold`     | Constitutional committee cold credential                              | committee cold credential          |
+
+#### Constitutional Committee Hot Keys
+
+Constitutional hot keys and credential should be encoded in Bech32 with the following prefixes:
+
+| Prefix       | Meaning                                                               | Contents                           |
+| ------------ | --------------------------------------------------------------------- | ---------------------------------- |
+| `cc_hot_sk`  | CIP-1852’s constitutional committee hot signing key                   | Ed25519 private key                |
+| `cc_hot_vk`  | CIP-1852’s constitutional committee verification signing key          | Ed25519 private key                |
+| `cc_hot_xsk` | CIP-1852’s constitutional committee hot extended signing key          | Ed25519-bip32 extended private key |
+| `cc_hot_xvk` | CIP-1852’s constitutional committee extended verification signing key | Ed25519 public key with chain code |
+| `cc_hot`     | Constitutional committee hot credential                               | committee hot credential           |
+
+### Tooling Definitions
+
+### DRep Keys
+
+Supporting tooling should clearly label these key pairs as "DRep Keys".
+
+Examples of acceptable `keyType`s for supporting tools:
+
+| `keyType`                                   | Description                    |
+| ------------------------------------------- | ------------------------------ |
+| `DRepSigningKey_ed25519`                    | DRep Signing Key               |
+| `DRepExtendedSigningKey_ed25519_bip32`      | DRep Extended Signing Key      |
+| `DRepVerificationKey_ed25519`               | DRep Verification Key          |
+| `DRepExtendedVerificationKey_ed25519_bip32` | DRep Extended Verification Key |
+
+For hardware implementations:
+
+| `keyType`                     | Description                    |
+| ----------------------------- | ------------------------------ |
+| `DRepHWSigningFile_ed25519`   | Hardware DRep Signing File     |
+| `DRepVerificationKey_ed25519` | Hardware DRep Verification Key |
+
+#### Constitutional Committee Cold Keys
+
+Supporting tooling should clearly label these key pairs as "Constitutional Committee Cold Keys".
+
+Examples of acceptable `keyType`s for supporting tools:
+
+| `keyType`                                                    | Description                                             |
+| ------------------------------------------------------------ | ------------------------------------------------------- |
+| `ConstitutionalCommitteeColdSigningKey_ed25519`              | Constitutional Committee Cold Signing Key               |
+| `ConstitutionalCommitteeColdExtendedSigningKey_ed25519`      | Constitutional Committee Cold Extended Signing Key      |
```

Comment:
if derived from the path, shouldn't all the extended keys - like with drep - be also marked with _bip32 ?

Reply:
Good spot, thanks.
Fixed in [9fe4a14](https://github.com/cardano-foundation/CIPs/pull/597/commits/9fe4a14517f0aee20a4782df198b0619a24e8461).